src/content/all/it-assistance.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "IT Assistance"
+---
+
+For all home access needs!
+
+<!--more-->
+
+## How to Download DoD Root Certificates
+
+To access DoD websites from your personal computer, you will need to install the DoD Root certificates. **InstallRoot** is a program created by the Defense Information Systems Agency (DISA) to automate the installation. Here are instructions for installing the program and downloading the certificates:
+
+**Note: This is for Windows systems only**
+
+ 1. Visit the [DoD Cyber Exchange](https://public.cyber.mil/pki-pke/pkipke-document-library/?_dl_facet_pkipke_type=tools) website (created by DISA) to download InstallRoot. 
+ 2. Scroll to the bottom of the page until you find "InstallRoot 5.X: NIPR Windows Installer". Most systems will be 64-bit.
+ 3. Click on the installer, save to your local computer, and run the program.
+ 4. Click "Next" all the way through and then click "Install".
+ 5. Once InstallRoot is installed, run the program. Click "Install Certificates" and then click "Ok".
+ 6. Restart Internet Explorer and try to open a DoD, CAC-enabled website.
+
+{{<figure src="/assets/img/content/install-root-install-certificates.jpg"class="align center small">}}
```

Comment:
There needs to be a space after `certificates.jpg"`.

Also, `align center` should be `aligncenter`.